rotkehlchen/accounting/accountant.py

```diff
@@ -228,6 +229,8 @@ def _process_event(
         or with reading the response returned by the server
         """
         ignored_assets = self.db.get_ignored_assets()
+        eth2_taxable_after_withdrawal_enabled = db_settings.eth2_taxable_after_withdrawal_enabled
```

Comment:
Why not accessing database setting directly in the condition?

Reply:
Does it make much of a difference? Or is there any reason, in particular, you're suggesting that?

Reply:
Nope, no reason. Just left a comment in case you would have wanted to change this. But no particular reason, so should be fine as is.